app/models/category_localization.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CategoryLocalization < ActiveRecord::Base
+  belongs_to :category
+
+  validates :locale, presence: true, length: { maximum: 20 }
```

Comment:
Since we know can get all the possible locales via `I18n.available_locales`, should we make this an enum column to limit the possible values of this column?